student-records-console-app/studentRecordsConsoleApp.cpp

```diff
@@ -0,0 +1,276 @@
+#pragma once
+
+#include <mongocxx/client.hpp>
+#include <bsoncxx/builder/stream/document.hpp>
+#include <bsoncxx/json.hpp>
+#include <mongocxx/uri.hpp>
+#include <mongocxx/instance.hpp>
+
+#include <algorithm>
+#include <iostream>
+#include <vector>
+
+using namespace std;
+// Add URI inside the curly braces in below line.  
+static const mongocxx::uri s_Cluster0_uri = mongocxx::uri{ /*Add Uri here ---> */""};

```

Comment:
I understand it would be a good practice to fetch the URI from an external source generally, but could you help me understand the rationale behind fetching it from env variable rather than just mentioning it here? Bearing in mind, this is a program meant for a new dev to get started asap. Adding a env variable into the mix adds one more step for them. There could be more barriers (for example, students working in labs where they may not have access to create new env variables).

Reply:
I understand that this is the simplest way (to type the URI in the code). However there are a number of instances in the past where new developers just copy the example code and ended up committing their Atlas URI on GitHub public repository. Some even think that this method is normal.

Reply:
I see, that's a fair enough reason. I'll update this.

Reply:
Done.
PS: `getenv` causes compilation error since MSVC flags it as an unsafe method. Using a safer alternative - `_dupenv_s`